JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -102,7 +104,7 @@ private fun PostHeaderImage(post: Post) {
     post.image?.let { image ->
         Container(modifier = LayoutHeight.Min(180.dp) + LayoutWidth.Fill) {
             Clip(shape = RoundedCornerShape(4.dp)) {
-                DrawImage(image)
+                SimpleImage(image = image)
```

Comment:
As `SimpleImage` is a layout node then we shouldn't need the outer container here?

Reply:
We don't have proper clipping so we will loose it with one layout node, it's doable to write it using draw modifiers, but will be less performant.

But anyway we probably need `modifier = LayoutSize.Fill` here to have same behaviour as previously

Reply:
Investigating.

```
Clip(shape = RoundedCornerShape(4.dp)) {
    val imageModifier = ImagePainter(image).toModifier(scaleFit = ScaleFit.FillHeight)
    Box(modifier = LayoutHeight.Min(180.dp) + LayoutWidth.Fill + imageModifier)
}
```

Is not clipping trying to understand why.

Reply:
Figured it out –

1. Clip needs to have a sized container that it's placed in.
2. Box needs to have it's dimensions explicitly specified.

This is all in flight so checking in a working fix for clipping.